extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-a4a-config.js

```diff
@@ -56,33 +58,32 @@ describe('doubleclick-a4a-config', () => {
   });
 
   describe('#doubleclickIsA4AEnabled', () => {
-    it('should enable a4a when requested and on CDN', () => {
-      mockWin.location = parseUrl(
-          'https://cdn.ampproject.org/some/path/to/content.html');
+    it('should enable a4a when native crypto is supported', () => {
+      forceExperimentBranch(mockWin, DFP_CANONICAL_FF_EXPERIMENT_NAME,
+          DOUBLECLICK_EXPERIMENT_FEATURE.CANONICAL_EXPERIMENT);
       const elem = testFixture.doc.createElement('div');
-      elem.setAttribute(BETA_ATTRIBUTE, 'true');
       testFixture.doc.body.appendChild(elem);
       expect(doubleclickIsA4AEnabled(mockWin, elem)).to.be.true;
-      expect(elem.getAttribute(EXPERIMENT_ATTRIBUTE)).to.equal(
-          BETA_EXPERIMENT_ID);
     });
 
-    it('should enable a4a when requested and in local dev', () => {
+    it('should not enable a4a when native crypto is not supported', () => {
+      const instance = new DoubleclickA4aEligibility();
+      sandbox.stub(instance, 'supportsCrypto', () => false);
```

Comment:
I'm surprised this works give you never pass instance to anything.  I would have expected:

sandbox.stub(DoubleclickA4aEligibility.prototype, 'supportsCrypto').returns(false);